src/customdocument.cpp

```diff
@@ -103,7 +106,7 @@ QString CustomDocument::getMarkdownUrlAtPosition(const QString &text, int positi
             const int foundPositionEnd = foundPositionStart + linkText.size();
 
             // check if position is in found string range
-            if ((position >= foundPositionStart) && (position <= foundPositionEnd)) {
+            if ((position >= foundPositionStart) && (position < foundPositionEnd)) {
```

Comment:
This fixes a bug where if a link is the last thing in a line, all whitespace to the right of the link will be clickable. Adding a space to the right of the link would be a temporary fix.

Maybe there's a better way to fix this?